CMakeLists.txt

```diff
@@ -32,26 +32,26 @@ option(NOVELRT_BUILD_INTEROP "Build NovelRT's Interop library" ON)
 option(NOVELRT_INCLUDE_CONAN_PATHS "Include conan_paths toolchain" ON)
 
 set(NOVELRT_BZIP2_VERSION "1.0.8" CACHE STRING "BZip2 version")
-set(NOVELRT_DOTNET_VERSION "6.0.100" CACHE STRING "Dotnet version")
 set(NOVELRT_DOXYGEN_VERSION "1.8.17" CACHE STRING "Doxygen version")
 set(NOVELRT_FLAC_VERSION "1.3.3" CACHE STRING "FLAC version")
 set(NOVELRT_FREETYPE_VERSION "2.10.1" CACHE STRING "Freetype version")
-set(NOVELRT_GLFW_VERSION "3.3.2" CACHE STRING "GLFW version")
+set(NOVELRT_GLFW_VERSION "3.3.6" CACHE STRING "GLFW version")
 set(NOVELRT_GLM_VERSION "0.9.9.7" CACHE STRING "GLM version")
 set(NOVELRT_GSL_VERSION "3.1.0" CACHE STRING "ms-gsl version")
 set(NOVELRT_GTEST_VERSION "1.10.0" CACHE STRING "GTest version")
-set(NOVELRT_OGG_VERSION "1.3.4" CACHE STRING "Ogg version")
+set(NOVELRT_OGG_VERSION "1.3.5" CACHE STRING "Ogg version")
 set(NOVELRT_ONETBB_VERSION "2021.3.0" CACHE STRING "OneTBB version")
 set(NOVELRT_OPENAL_VERSION "1.21.1" CACHE STRING "OpenAL version")
 set(NOVELRT_OPUS_VERSION "1.3.1" CACHE STRING "Opus version")
 set(NOVELRT_PNG_VERSION "1.6.37" CACHE STRING "PNG version")
-set(NOVELRT_SNDFILE_VERSION "1.0.29" CACHE STRING "Sndfile version")
-set(NOVELRT_SPDLOG_VERSION "1.5.0" CACHE STRING "Spdlog version")
+set(NOVELRT_SNDFILE_VERSION "1.0.30" CACHE STRING "Sndfile version")
+set(NOVELRT_SPDLOG_VERSION "1.8.2" CACHE STRING "Spdlog version")
 set(NOVELRT_VORBIS_VERSION "1.3.7" CACHE STRING "Opus version")
-set(NOVELRT_VULKAN_VERSION "1.2.172" CACHE STRING "Vulkan version")
+set(NOVELRT_VULKAN_VERSION "1.2.198" CACHE STRING "Vulkan version")
 set(NOVELRT_VULKAN_MEMORY_ALLOCATOR_VERSION "2.3.0" CACHE STRING "VulkanMemoryAllocator version")
 
 if (NOVELRT_INCLUDE_CONAN_PATHS)
+  set(CONAN_CMAKE_SILENT_OUTPUT ON)
```

Comment:
Just a reminder - this hushes Conan from spewing a lot of verbose information we don't need via CMake